src/main.py

```diff
@@ -36,6 +37,15 @@
     }
 )
 
+origins = ["*"]
+
+api.add_middleware(
+    CORSMiddleware,
+    allow_origins=origins,
+    allow_credentials=True,
+    allow_methods=["GET"],
+)
+
```

Comment:
Could you please add a comment, refactor out the `origins` variable, and remove the extra newline between this code and the `include_router` call?
```suggestion
# Add middleware
api.add_middleware(
    CORSMiddleware,
    allow_origins=["*"],
    allow_credentials=True,
    allow_methods=["GET"],
)
```